esmvaltool/preprocessor/_derive/lvp.py

```diff
@@ -0,0 +1,39 @@
+"""Derivation of variable `lvp`.
+
+authors:
+    - weig_ka
+
+"""
+from iris import Constraint
+
+from ._baseclass import DerivedVariableBase
+
+
+class DerivedVariable(DerivedVariableBase):
+    """Derivation of variable `lvp`."""
+
+    # Required variables
+    required = [
+        {
+            'short_name': 'hfls'
+        },
+        {
+            'short_name': 'pr'
+        },
+        {
+            'short_name': 'evspsbl'
+        },
+    ]
```

Comment:
```suggestion
required = [
        { 'short_name': 'hfls'},
        { 'short_name': 'pr' },
        {'short_name': 'evspsbl'},
    ]
```

Reply:
Ok, I removed the lines in all 6 preprocessor/_derive/*.py files